Tests/Common/Exceptions/NoMethodMatchPythonExceptionInterpreterTests.cs

```diff
@@ -56,7 +56,7 @@ public void Setup()
         [TestCase(typeof(KeyNotFoundException), ExpectedResult = false)]
         [TestCase(typeof(DivideByZeroException), ExpectedResult = false)]
         [TestCase(typeof(InvalidOperationException), ExpectedResult = false)]
-        [TestCase(typeof(PythonException), ExpectedResult = true)]
+        [TestCase(typeof(PythonException), ExpectedResult = false)]
         public bool CanInterpretReturnsTrueForOnlyNoMethodMatchPythonExceptionType(Type exceptionType)
```

Comment:
Isn't this test method name misleading? I don't see a `NoMethodMatchPythonException` test case?

Reply:
Agree, but it follows the pattern of the other tests.
Can we changed for `CanInterpretReturnsFalseForAnyGivenExceptionType`? Any suggestion?